flink-runtime-web/web-dashboard/app/scripts/modules/jobs/jobs.svc.coffee

```diff
@@ -216,4 +216,7 @@ angular.module('flinkApp')
     # proper "DELETE jobs/<jobid>/"
     $http.get "jobs/" + jobid + "/yarn-cancel"
 
+  @stopJob = (jobid) ->
+    $http.delete "jobs/" + jobid + "/stop"
```

Comment:
As I commented before, this should rather be a `GET` request at `/yarn-stop` to support Yarn cancellations.